docs/plans/2026-01-18-brainstorming-pipeline-design.md

```
@@ -0,0 +1,421 @@
+
+CREATE INDEX idx_brainstorm_artifacts_session ON brainstorm_artifacts(session_id);
+```
+
+---
+
+## API Endpoints
+
+### Session Lifecycle
+
+```
+POST   /api/brainstorm/sessions              → Create new session
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)